compiler/stable_mir/src/mir/mono.rs

```diff
@@ -142,6 +147,7 @@ impl Debug for Instance {
         f.debug_struct("Instance")
             .field("kind", &self.kind)
             .field("def", &self.mangled_name())
+            .field("args", &self.args())
```

Comment:
In the future, please pull unrelated changes at least into a separate commit, but ideally into a separate PR. This has nothing to do with what's mentioned in the title of the PR, which is a one-liner 😓

Reply:
No worries, just surprising to see so many changes when I expected a one-liner 😆

Reply:
I created https://github.com/rust-lang/rust/pull/119141 for that, and I'll remove it from here.